README.md

```diff
@@ -1,53 +1,96 @@
-a stupid cat programmer's crazy vencord plugins
+`😺` A stupid cat programmer's crazy Vencord plugins.
 
-join our [discord server](https://discord.gg/PxDj9XeHkB) for more info
+`🗨️` Join our [Discord Server](https://discord.gg/PxDj9XeHkB) for more information.
 
-# FAQ
-## Is this original Vencord?
-No, this is a fork of Vencord with unofficial plugins. You can find the original Vencord [here](https://github.com/Vendicated/Vencord).
+# Frequently Asked Questions
+
+## Is this the original Vencord client modification?
+No, this is a fork of Vencord with unofficial plugins. You can find the original repository of Vencord [here](https://github.com/Vendicated/Vencord).
+
+---
 
 ## Is this fork safe?
-Yes but if you don't trust, you can check the source code or ask someone professional to check it for you.
+Yes, this repository is safe. If you don't trust it, feel free to look at the source code.
 
-## Is this fork up-to-date with original Vencord?
-Yes but we don't update the fork immediately after the original Vencord is updated and you will need to update it manually. We will release a new version when there is necessary changes.
+---
 
-# How to use?
+## Is this fork up-to-date with original Vencord?
+Yes, it usually is, but we don't update this fork immediatley after the original Vencord is updated. You'll have to update it manually. We release a new version when there are necessary changes.
 
-If you want to use these plugins in your Discord client, please follow the instructions below or watch [this tutorial](https://drive.google.com/file/d/1zlWv4t14ORqy7QuF_Sup7DLkf7pAgS90/view?usp=sharing).
+---
 
+# Dependencies
 1. Download the latest version of Node.js from [here](https://nodejs.org/en/download/current).
```

Comment:
If they are not steps, this could be better
```suggestion
 - Download the latest version of Node.js from [here](https://nodejs.org/en/download/current).
```